plaso/parsers/plist_plugins/macos_background_items.py

```diff
@@ -0,0 +1,273 @@
+        volume_mount_point = data_record.string
+        if relative_target_path:
+          relative_target_path = ''.join([
+              volume_mount_point, relative_target_path])
+
+        event_data.volume_mount_point = volume_mount_point
+
+      elif tagged_value.value_tag == 0x00002010:
+        event_data.volume_name = data_record.string
+
+      elif tagged_value.value_tag == 0x00002013:
+        data_type_map = self._GetDataTypeMap('float64be')
+        cocoa_timestamp = self._ReadStructureFromByteStream(
+            data_record.data, 0, data_type_map)
+
+        event_data.volume_creation_time = dfdatetime_cocoa_time.CocoaTime(
+            timestamp=cocoa_timestamp)
+
+      elif tagged_value.value_tag == 0x00002020:
+        data_type_map = self._GetDataTypeMap('bookmark_data_property_flags')
+        property_flags = self._ReadStructureFromByteStream(
+            data_record.data, 0, data_type_map)
+
+        event_data.volume_flags = (
+            property_flags.flags & property_flags.valid_flags_bitmask)
+
+      elif tagged_value.value_tag == 0x0000f017:
+        event_data.name = data_record.string
+
+    if relative_target_path:
+      event_data.target_path = relative_target_path
+
+  def _ParseIntegersArray(
+      self, bookmark_data, data_area_offset, data_record_offsets):
+    """Parses an integers array.
+
+    Args:
+      bookmark_data (bytes): bookmark data.
+      data_area_offset (int): offset of the data area relative to the start of
+          the file.
+      data_record_offsets (list[int]): offsets of the entry data records.
+
+    Returns:
+      list[int]: integers.
+    """
+    data_type_map = self._GetDataTypeMap('bookmark_data_record')
+
+    integers_array = []
+
+    for data_record_offset in data_record_offsets:
+      data_record_offset += data_area_offset
+      data_record = self._ReadStructureFromByteStream(
+          bookmark_data[data_record_offset:], data_record_offset,
+          data_type_map)
+
+      integers_array.append(data_record.integer)
+
+    return integers_array
+
+  def _ParseStringsArray(
+      self, bookmark_data, data_area_offset, data_record_offsets):
+    """Parses a strings array.
+
+    Args:
+      bookmark_data (bytes): bookmark data.
+      data_area_offset (int): offset of the data area relative to the start of
+          the file.
+      data_record_offsets (list[int]): offsets of the entry data records.
+
+    Returns:
+      list[str]: strings.
+    """
+    data_type_map = self._GetDataTypeMap('bookmark_data_record')
+
+    strings_array = []
+
+    for data_record_offset in data_record_offsets:
+      data_record_offset += data_area_offset
+      data_record = self._ReadStructureFromByteStream(
+          bookmark_data[data_record_offset:], data_record_offset,
+          data_type_map)
+
+      strings_array.append(data_record.string)
+
+    return strings_array
+
+
+class MacOS1013BackgroundItemsPlugin(MacOSBackgroundItemsPlistPlugin):
```

Comment:
MacOS1013BackgroundItemsPlugin and MacOS13BackgroundItemsPlugin can likely be combined into 1 class.